### PR TITLE
MWPW-153144 Pre-load Marquee image if the marquee is getting replaced in MEP

### PR DESCRIPTION
**Pull request by suhjainadobe**
Pre-load Marquee image if the marquee is getting replaced in MEP

Resolves: [MWPW-153144](https://jira.corp.adobe.com/browse/MWPW-NUMBER)

**Test URLs:**
- Before: https://main--cc--adobecom.hlx.page/drafts/suhjain/personalization/document
- After: https://eagerloadimg--cc--adobecom.hlx.live/drafts/suhjain/personalization/document?mep=off&martech=off

Screenshot of performance improvement:
Before:
image
After:
image

### CODE REVIEW

File: creativecloud/scripts/utils.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ function getDecorateAreaFn() {
 
   return (area, options) => {
     if (isRootPage()) replaceDotMedia();
-    if (!lcpImgSet) loadLCPImage(area, options);
+    loadLCPImage(area, options);
```
Comment (aishwaryamathuria) [inline creativecloud/scripts/utils.js:111]: can we change it to check if options have fragment link and if (!lcpImgSet) only then call loadLCPImage

File: creativecloud/scripts/utils.js [hunk 2]
```diff
@@ -78,7 +75,10 @@ function getDecorateAreaFn() {
   }
 
   async function loadLCPImage(area = document, { fragmentLink = null } = {}) {
-    const firstBlock = area.querySelector('body > main > div > div');
+    let firstBlock = area.querySelector('body > main > div > div');
+    if (!firstBlock && window.document.querySelector('body > div > div > a.fragment')) {
```
Comment (aishwaryamathuria) [inline creativecloud/scripts/utils.js:79]: const firstBlock = (!!fragmentLink) ? area.querySelector('body > div > div') || area.querySelector('body > main > div > div');

**Review by aishwaryamathuria — CHANGES_REQUESTED** [review]

File: creativecloud/scripts/utils.js (hunk 1 above)
Comment (suhjainadobe) [reply creativecloud/scripts/utils.js:111]: The same check anyways I am doing in loadLCPImage function at the start itself. But, ill can add them here as well

**Review by aishwaryamathuria — APPROVED** [review]

**Review by aishwaryamathuria — APPROVED** [review]

**Comment by spadmasa** [comment]
Validation can  be done once the PR is merged to stage

**Review by drashti1712 — APPROVED** [review]

**Pull request merged**